CHANGELOG.md

```diff
@@ -6,71 +6,72 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/)
 
 The types of changes are:
 
-* `Added` for new features.
```

Comment:
that MD lint though 😆

Reply:
ugh oops didn't see that, I can undo it?

Reply:
I decided to put the CHANGELOG.md in the .prettierignore

Reply:
oh it didn't actually bother me, I just assumed you had a VS code linter for yaml or something lol